lib/slack/web/config.rb

```diff
@@ -1,3 +1,4 @@
+require 'openssl'
```

Comment:
This would make OpenSSL hard dependency. 

Might be better to wrap it in a block and rescue `LoadError`.